### PR TITLE
Fr us 15

### LINKED ISSUE #18: User Story 15, Users who are logged in already are redirected
As a registered user, merchant, or admin
When I visit the login path
If I am a regular user, I am redirected to my profile page
If I am a merchant user, I am redirected to my merchant dashboard page
If I am an admin user, I am redirected to my admin dashboard page
And I see a flash message that tells me I am already logged in

### PR DESCRIPTION
**Pull request by fredrondina**
completed us 15 plus tests. auto redirects to designated landing page if you visit /login when already authenticated

### CODE REVIEW

File: spec/features/sessions/new_spec.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@
       expect(current_path).to eq("/merchant/dashboard")
     end
 
-    it "admin redirect to admin dashboard" do
+    it "admin redirect to admin dashboard upon login" do
```
Comment (kelshadarby) [inline spec/features/sessions/new_spec.rb:63]: Very clear, nice!

**Comment by fredrondina** [comment]
closes #18

**Review by brian-greeson — APPROVED** [review]
Looks good to me

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class SessionsController < ApplicationController
   def new
-
+    dynamic_redirect unless current_user.nil?
```
Comment (kelshadarby) [inline app/controllers/sessions_controller.rb:3]: This is awesome! 🥳

**Review by kelshadarby — APPROVED** [review]

**Pull request merged**